docs/modules/deployment/pages/core/install-jdbc-driver.adoc

```diff
@@ -0,0 +1,27 @@
+[[install-jdbc-driver-core]]
+= Install JDBC Driver on OpenNMS {page-component-title} Core
+
+To detect, poll, or collect any JDBC service, you must install the appropriate JDBC driver.
+
+. Download the JDBC driver JAR file from a repository (for example, a public https://mvnrepository.com/artifact/mysql/mysql-connector-java/8.0.22[Maven repository]) or from the database vendor.
+. Install the JAR file in \$OPENNMS_HOME/lib/
+. Restart {page-component-title}
+
+Testing an installed JDBC Driver
+.Connect to the Karaf shell
+[source,console]
+----
+ssh -p 8101 admin@localhost
+----
+
+.Attempt an ad-hoc collection using the database driver
```

Comment:
```suggestion
.Attempt an ad hoc collection using the database driver
```
Inspired by our recent style guide foray in to Latin, I googled "ad hoc" and discovered that it doesn't need to be hyphenated, even when it's a modifier.